setup.py

```diff
@@ -2,7 +2,8 @@
 from setuptools import setup
 
 setup(name='pyforce',
-    version='1.7.3', # be sure to update the version in pyforce.py too
+    version='1.8.0',
+    install_requires=['defusedxml==0.5.0'],
```

Comment:
This specific version, or anything higher than it?

Reply:
Good catch, added `defusedxml>=0.5.0`